backends/advanced/src/advanced_omi_backend/prompt_registry.py

```
@@ -0,0 +1,121 @@
+        template_text = None
+
+        # Try LangFuse first
+        try:
+            client = self._get_client()
+            if client is not None:
+                fallback = self._defaults.get(prompt_id, "")
+                prompt_obj = client.get_prompt(prompt_id, fallback=fallback)
+                if variables:
+                    return prompt_obj.compile(**variables)
+                return prompt_obj.compile()
+        except Exception as e:
```

Ruff (0.14.14)

[warning] Do not catch blind exception: `Exception`

(BLE001)